lib/pacemaker/libpacemaker_private.h

```diff
@@ -453,9 +453,11 @@ enum pcmk__coloc_affects {
 };
 
 G_GNUC_INTERNAL
-enum pcmk__coloc_affects pcmk__colocation_affects(const pe_resource_t *dependent,
+enum pcmk__coloc_affects pcmk__colocation_affects(const pe_resource_t
+                                                    *dependent,
```

Comment:
Could just as well put the return type on a separate line (like in the definition) when it gets this long, although that would add a tiny bit of noise to greps. I don't have strong feelings and wouldn't change this now.

Reply:
That does sound better, but not going to bother with this PR